hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -469,3 +476,332 @@ def hpat_pandas_series_floordiv_number_impl(lhs, rhs):
         return hpat_pandas_series_floordiv_number_impl
 
     raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
+
+@overload_method(SeriesType, 'pow')
+def hpat_pandas_series_pow(lhs, rhs):
+    """
+    Pandas Series method 'pow' implementation.
+        https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.pow.html#pandas.Series.pow
+
+    Algorithm: result = lhs.pow(other, level=None, fill_value=None, axis=0)
+
+    Where:
+               lhs: pandas.series
+             other: pandas.series or scalar value
+             level: unsupported
+        fill_value: unsupported
+              axis: unsupported
+            result: pandas.series
+
+    Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5
+    """
+
+    _func_name = 'Method pow().'
+
+    if not isinstance(lhs, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given lhs: {}'.format(_func_name, lhs))
+
+    if isinstance(rhs, SeriesType):
+        def hpat_pandas_series_pow_impl(lhs, rhs):
+            """
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5
+            """
+
+            return pandas.Series(lhs._data ** rhs._data)
+
+        return hpat_pandas_series_pow_impl
+
+    if isinstance(rhs, types.Integer) or isinstance(rhs, types.Float):
+        def hpat_pandas_series_pow_impl(lhs, rhs):
+            """
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5_integer_scalar
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5_float_scalar
+            """
+
+            return pandas.Series(lhs._data ** rhs)
+
+        return hpat_pandas_series_pow_impl
+
+    raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
```

Comment:
The object must be pandas.series and argiment must be a number. Given lhs:... rhs...